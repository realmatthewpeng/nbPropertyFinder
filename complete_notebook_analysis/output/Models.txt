Total number of lines found: 49

df_county_data
state=df_county_data['State']
grad_rate = df_county_data["Graduation Rate"]
county = df_county_data["County Name"]
pov_rate = df_county_data["Poverty Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
pov_rate = df_county_data["Poverty Rate"]
pov_rate = df_county_data["Poverty Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
x = df_county_data["Poverty Rate"]
y = df_county_data["Graduation Rate"]
house_size = df_county_data["Household Size"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Household Size"]
county = df_county_data["County Name"]
pov_rate = df_county_data["Household Size"]
grad_rate = df_county_data["Graduation Rate"]
x = df_county_data["Household Size"]
y = df_county_data["Graduation Rate"]
house_size = df_county_data["Unemployment Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Unemployment Rate"]
county = df_county_data["County Name"]
pov_rate = df_county_data["Unemployment Rate"]
grad_rate = df_county_data["Graduation Rate"]
x = df_county_data["Unemployment Rate"]
y = df_county_data["Graduation Rate"]
house_size = df_county_data["Median Income"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Median Income"]
pov_rate = df_county_data["Median Income"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
x = df_county_data["Median Income"]
y = df_county_data["Graduation Rate"]
house_size = df_county_data["Speak a language other than English"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Speak a language other than English"]
county = df_county_data["County Name"]
pov_rate = df_county_data["Speak a language other than English"]
grad_rate = df_county_data["Graduation Rate"]
x = df_county_data["Speak a language other than English"]
y = df_county_data["Graduation Rate"]

Incorrect Lines:


Precision = 1.0
Recall = 1.0

