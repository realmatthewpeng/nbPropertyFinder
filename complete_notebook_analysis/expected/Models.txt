state=df_county_data['State']
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
pov_rate = df_county_data["Poverty Rate"]
x = df_county_data["Poverty Rate"]
y = df_county_data["Graduation Rate"]
house_size = df_county_data["Household Size"]
pov_rate = df_county_data["Household Size"]
x = df_county_data["Household Size"]
house_size = df_county_data["Unemployment Rate"]
pov_rate = df_county_data["Unemployment Rate"]
x = df_county_data["Unemployment Rate"]
house_size = df_county_data["Median Income"]
pov_rate = df_county_data["Median Income"]
x = df_county_data["Median Income"]
house_size = df_county_data["Speak a language other than English"]
df_county_data
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
pov_rate = df_county_data["Poverty Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
pov_rate = df_county_data["Poverty Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Household Size"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
y = df_county_data["Graduation Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Unemployment Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
y = df_county_data["Graduation Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Median Income"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
y = df_county_data["Graduation Rate"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
house_size = df_county_data["Speak a language other than English"]
county = df_county_data["County Name"]
grad_rate = df_county_data["Graduation Rate"]
pov_rate = df_county_data["Speak a language other than English"]
y = df_county_data["Graduation Rate"]
x = df_county_data["Speak a language other than English"]